workspace/src/github.com/darkcrux/go-ceph-rest-api/handler_pool.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+  "fmt"
+  "net/http"
+  "github.com/gorilla/mux"
+  "encoding/json"
+)
+
+const (
+  // actions
+  create = "create"
+  // get = "get"
+  // delete = "delete"
+
+  // methods
+  // get = "GET"
+  // post = "POST"
+  put = "PUT"
+  // delete = "DELETE"
+)
+
+type PoolAPI struct {
+  Res http.ResponseWriter
+  Req *http.Request
+  Params map[string]string
+}
+
+type Response struct {
+  Status string      `json:"status"`
+  Output interface{} `json:"output"`
+}
+
+func poolHandler(res http.ResponseWriter, req *http.Request) {
+  res.WriteHeader(200)
+  var pool PoolAPI
+  pool.Res = res
+  pool.Req = req
```

Comment:
This can be rewritten as:

```
pool := PoolAPI {
    Res: res,
    Req: req,
}
```

or 

pool := &PoolAPI{
    Res: res,
    Req: req,
}

would be more concise.